.github/workflows/build-packages.yml

```diff
@@ -0,0 +1,33 @@
+name: Build packages
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Install dependencies
+        run: |
+          npm ci
+          npm ci --prefix packages/embed
+          npm ci --prefix packages/embed
+
+      - name: Build @excalidraw/excalidraw
+        run: |
+          npm run pack --prefix packages/embed
```

Comment:
Can we rename this to `packages/excalidraw` since that's what the name of the package is excluding the namespace ?